selfdrive/ui/qt/onroad.h

```diff
@@ -2,13 +2,16 @@
 
 #include <map>
 
+#include <QOpenGLFunctions>
+#include <QOpenGLWidget>
 #include <QSoundEffect>
-#include <QtWidgets>
```

Comment:
does this actually make a difference? anything unused should get optimized out

Reply:
From what I understand, compiler will read all includes and  "insert" their content  to the source file,then do the compile and optimize steps.it's our duty to cleanup the includes